Cargo.toml

```diff
@@ -14,6 +14,7 @@ actix = "0.11"
 anyhow = "1.0"
 cfg-if = "1.0"
 chrono = { version = "0.4", features = ["serde"] }
+ctrlc = { version = "3.0", features = ["termination"] }
```

Comment:
I think it'd be more natural to use https://docs.rs/tokio/1.5.0/tokio/signal/index.html